File: mobile/SeethroughApp/app/src/main/java/com/example/seethroughapp/data/model/meal/Meal.java
package com.example.seethroughapp.data.model.meal;

import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter @Setter
@ToString
public class Meal {
    private String mealId;
    private String memberId;
    private String servingDate;
    private String servingTime;
    private List<String> menu;
    private String reason;
}

File: client/src/interfaces/Meal.ts
export interface Meal {
  meal_id: string;
  member_id: string;
  serving_date: string;
  serving_time: "아침" | "점심" | "저녁";
  menu: string[];
  reason: string;
}

export interface MealPlanResponse {
  breakfast: Meal;
  lunch: Meal;
  dinner: Meal;
}
